svc/rewards/repository/sql/migrations/20210602072607-create_rewards_table.sql

```diff
@@ -11,7 +11,8 @@ $$ LANGUAGE 'plpgsql';
 CREATE TABLE IF NOT EXISTS rewards (
     id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
     user_id uuid NOT NULL,
-    quiz_id uuid DEFAULT NULL,
+    relation_id uuid DEFAULT NULL,
```

Comment:
You cannot change an existed migration file. You must add a new one. Look at this svc/rewards/repository/sql/migrations/20210614091403-add_transaction_type_field_to_rewards_table.sql